### PR TITLE
Marley's Cake Adventure

### PR DESCRIPTION
**Pull request by kalez43**
This is a text-based game, it is a work in progress and I will add more as I go along.

### CODE REVIEW

File: marley_cake_adventure.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+print("Kayleigh places the cake on the counter and walks 20 feet away to sit down on the couch.")
+space()
+print("""You notice that she is facing away from the counter, and is reading a book. Her attention should be 
+filled for now.""")
+enter()
+stars()
+
+
+def decision1():
+    print("""You can:
```
Comment (Bazarovay) [inline marley_cake_adventure.py]: You are making good progress, using methods and all! Great job! If I were to suggest anything, I'd say remove the 1 from the method name. This from the book Clean Code: https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882

_Methods should have verb or verb phrase names like postPayment, deletePage, or save.
Accessors, mutators, and predicates should be named for their value and prefixed with get,
set, and is according to the javabean standard._ - Clean Code

File: marley_cake_adventure.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+def stars():
+    space()
+    print("* * * * *")
+    space()
+
+
+# Enter function between story telling strings
+def enter():
+    space()
```
Comment (Bazarovay) [inline marley_cake_adventure.py]: Won't this while loop end after the first iteration?

Comment (kalez43) [reply marley_cake_adventure.py]: Yes, I wanted a way for the user to have to press enter in order to move to the next sequence

File: marley_cake_adventure.py (hunk 1 above)
Comment (kalez43) [reply marley_cake_adventure.py]: Wait, which part is a method?? decision1()? I still don't fully understand the terminology :/

Comment (Bazarovay) [reply marley_cake_adventure.py]: Yes. Method == function

**Comment by Bazarovay** [comment]
Please resolve these so that I can merge the request

**Pull request closed without merging**